libs/lib_parsing/Pos.mli

```diff
@@ -34,6 +34,13 @@ val string_of_pos : t -> string
 (* Adjust line x col in a position *)
 (*****************************************************************************)
 
+(*
+   Return (line, column) from a byte position.
+
+   If the byte position is out of range, the functions of this type return
+   the first position or the last position in the file.
```

Comment:
it returns the first or the last? I'm confused.

Reply:
The nearest one. I updated the comment.